mxnet-bot/LabelBotAddLabels/serverless.yml

```diff
@@ -0,0 +1,49 @@
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# Configurations
+
+service: LabelBot
+
+provider:
+  name: aws
+  runtime: python3.6
+  timeout: 180
+  iamRoleStatements:
+    -  Effect: "Allow"
+       Action:
+         - "secretsmanager:GetSecretValue"
+         - "secretsmanager:DescribeSecret"
+       # replace resource with your secret's ARN
+       # Resource: "arn:aws:secretsmanager:{region_name}:{account_id}:secret:{secret_id}"
+       Resource: "arn:aws:secretsmanager:us-east-1:072971575220:secret:credentials/cathy_github-3UiA11"
```

Comment:
Do I still have the access to this credential? please remove it.

Reply:
No, other people don't have the access to this secret. This is secret's ARN is more like an ID.

Reply:
fixed it